config/samples/example-test.sh

```diff
@@ -485,4 +485,9 @@ waitComponentPlanRetryTime "default" "do-once-nginx-sample-15.0.2" "5"
 info "5.7.4 verify this componentplan will failed, show error log"
 kubectl get cpl do-once-nginx-sample-15.0.2 --output='jsonpath={.status.conditions[?(@.type=="Actioned")]}'
 
+info "5.8 Verify that helm add repo with basic_auth"
+kubectl apply -f config/samples/core_v1alpha1_componentplanchartmuseum.yaml
+waitComponentPlanDone "kubebb-system" "chartmuesum-3.10.1"
+kubectl apply -f config/samples/core_v1alpha1_repository_chartmuseum.yaml
+
```

Comment:
IMO, Missing some validation steps, just adding the repo here, but as long as the repo doesn't have yaml formatting errors, kubectl apply always succeeds. 

I guess in the end we need to actually install one of the components in this chartmuseum. 
But there is no component in the default chartmuseum, so we need to upload a chart package.

Upload it via curl or https://github.com/chartmuseum/helm-push plugin and expose the chart museum via kube proxy or ingress.

Reply:
Yes,that means I need to create mychart package locally and push it.
That also means I need to commit the package to the branch.

Reply:
the test-chart.tgz can just use online file, such as https://charts.bitnami.com/bitnami/nginx-15.1.2.tgz

What do you guys think? @0xff-dev @bjwswang

Reply:
If chartmuseum adds authentication, I think it can be determined whether the function is normal by directly checking whether curl returns a status code of 200.

```shell
# expect 200
curl -u"foo:bar" -s -o /dev/null -w "%{http_code}" http://ip:port/index.yaml

# expect 401
curl -s -o /dev/null -w "%{http_code}" http://ip:port/index.yaml
```

If the output is 200, it means normal, and 401 means authentication failed.

Reply:
> it can be determined whether the function is normal by directly checking whether curl returns a status code of 200.

That's it. 

But we still need to verify whether the `helm install` part is normal when there is basic auth.